apps/sync/js/sync-engine/syncengine.js

```diff
@@ -106,6 +106,7 @@ var SyncEngine = (function() {
             payloadEnc => {
           return {
             id: record.id,
+            last_modified: record.last_modified,
```

Comment:
We should never upload `.last_modified`, it is assigned server-side. See https://docs.services.mozilla.com/storage/apis-1.5.html#concurrency-and-conflict-management